### ISSUE TITLE
is the project dead?

### ISSUE THREAD

**Issue by str** [victorporof/Tilt#26]
is the project dead?

**Comment by Taz8du29** [comment]
yes, totally ! Mozilla even removed the native implementation they did from DevTools :/

**Comment by str** [comment]
Could it be moved to a plugin?

**Comment by Taz8du29** [comment]
It can, but you'll have to convert the whole thing to the new extension system of FF Quantum.
If you plan to do it, good luck!

**Comment by ecotechie** [comment]
@victorporof any thoughts as to why this tool was removed form Firefox? And how much work it would be to make it into an add-on again?

**Comment by Bilge** [comment]
R.I.P. in piece.

**Comment by victorporof** [comment]
💀

**Comment by s-light** [comment]
if anybody knows a currently working alternative for Firefox please let us know!

**Comment by Davilink** [comment]
> if anybody knows a currently working alternative for Firefox please let us know!

MS Edge has this built-in. Sadly we are at the time that MS Edge is better than FF for web development, and this a fu.... low for mozilla, and i'm a daily driver of Firefox, so... yeah, Mozilla drop the ball solid, the only thing that still keep on FF is the Mutli-Account container (and no, profile aren't the same).

**Comment by risingfish** [comment]
As former user of this plugin I just want to say thank you @victorporof . It was an incredible tool that really was indispensable for building well structured  pages. I'm still doing web development today and I just shake my head seeing the markup that front libraries like React, Vue, and Angular spit out. Thanks for the fond memories!
